presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -876,4 +897,18 @@ Hive Connector Limitations
 
 * :doc:`/sql/delete` is only supported if the ``WHERE`` clause matches entire partitions.
 * :doc:`/sql/alter-schema` usage fails, since the Hive metastore does not support renaming schemas.
+* Writing to and creation of transactional tables is not supported.
+* Reading ORC ACID tables created with `Hive Streaming Ingest <https://cwiki.apache.org/confluence/display/Hive/Streaming+Data+Ingest>`_ are not supported.
+
+Hive 3 Related Limitations
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+* For security reasons, the ``sys`` system catalog is not accessible.
+* Hive's ``timestamp with local zone`` data type is not supported.
+  It is possible to read from a table with a column of this type, but the column
+  data is not accessible. Writing to such a table is not supported.
+* Due to Hive issues `HIVE-21002 <https://issues.apache.org/jira/browse/HIVE-21002>`_,
+  `HIVE-22167 <https://issues.apache.org/jira/browse/HIVE-22167>`_, Presto does
+  not correctly read ``timestamp`` values from Parquet, RCFile with the binary serde and
```

Comment:
Let's call this "RCBinary" like we do above in the supported file types section.